department-of-veterans-affairs/abd-vro
# Decouple container image versions

## What was the problem?

When deploying, we can only specify a single version for all container images.

Associated tickets or Slack threads:
- #1725

## How does this fix it?


Enable specifying versions of individual container images for deployment.
Versions specified in `scripts/image_versions.src` will be used for deployment. If no version is specified the `latest` is used.

To be done in separate PR:
- create GH Action workflows to pin and unpin image versions
- present image versions as options for manual runs of `Update Deployment` GitHub Action workflow

## How to test this PR
Run [Update Deployment action](https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/workflows/update-deployment.yml) and examine LHDI for expected result.
